language-design.md

```diff
@@ -34,6 +49,8 @@ How might the themes of _Growing a Language_ relate to ideas from the Fowler rea
 
 **Response**
 
+    I think they're both concerned with the design of programming languages and to make a set of guiding principles when designing them. For instance, Steele is concerned with readability as "programming languages need to be more like the languages we speak—but it might be good, too, if we were to use the languages we speak more in the way that we now use programming languages."[Steele, 1998], and Fowler mentions that "The overall goal for a DSL, as with any writing, is clarity for the reader."[Fowler, 2010]. The interesting part though is that Fowler also mentions that designers shouldn't "try to make the DSL read like natural language"[Fowler, 2010]. I think this could infer that Steele is concerned with making programming languages sound more like natural language conventions, whereas Fowler is concerned with making languages (or more specifically DSL's) more precise and readable.
```

Comment:
This is definitely an interesting debate on weather or not a programming language should read like a language we speak. I personally think that if a language is easily readable and writable then it shouldn't matter if it is similar to how we speak.